README.md

```diff
@@ -53,24 +55,31 @@ git pull
 
 ## Configuration
 
-To use this module, add it to the modules array in the `config/config.js` file.
+To use this module, add a configuration to the modules array in the `config/config.js` file.
+
+*Note*: Some of the module names used in the following examples are fictitious.
+
+### Module name based configuration
 
-*Note*: Some of the module names used in the following example are fictitious.
+The easiest way to configure this module is to use the module names to define on which page they should appear:
+
+The first element of the array is the first page, the second element is the second page, and so on.
 
 ```js
     {
         module: "MMM-pages",
         config: {
+            rotationTime: 1000 * 20,         // rotate every 20 seconds
             modules: [
-                ["newsfeed"],
-                ["calendar", "compliments"]
+                ["newsfeed"],                // page 1
+                ["calendar", "compliments"], // page 2
             ],
-            fixed: [
+            fixed: [                         // modules that are always shown
                 "clock",
                 "weather",
                 "MMM-page-indicator"
             ],
-            hiddenPages: {
+            hiddenPages: {                   // modules that are only shown on specific pages
```

Comment:
The comments here are aligned but the comments below aren't aligned?

Reply:
Just fixed.